sonic-net/sonic-buildimage
# [Supermicro-SSE-G3748] Add new device, platform to support SSE-G3748 p…

…latform.



#### Why I did it
Add new device and platform files to support Supermicro SSE-G3748 1G switch.
The switch is based on Marvell AC5X ASIC. 

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

- Create a directory arm64-supermicro_sse_g3748-r0 in ./device/supermicro. Put required device files in there.  Create another subdirectory using hwsku name supermicro_sse-g3748 in ./device/supermicro/arm64-supermicro_sse_g3748-r0 to store EZB v1.06 profiles for SAI configuration.
-  Create a directory sonic-platform-supermicro in ./platform/marvell/ to include all of requirement files and drivers to allow image to be able to install and work properly on SSE-G3748 platform.
- Modify files one-image.mk, rules.mk, sonic_fit.its and platform_arm64.conf so that the build can include building sonic-platform-supermicro-sse-g3748_1.0_arm64.deb for image. Ensure image installation and running properly for hardware platform.


#### How to verify it
Build the image for the platform by using below command:
    cd  sonic-buildimage
    make init
    make configure PLATFORM=marvell PLATFORM_ARCG=arm64
    make target/sonic-marvell-arm64.bin
Install and run the image in SSE-G3748 switch platform.
Verify by using command "show platform summary", "show platform fan", show interface status" .... to see if it works properly.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)
master



#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
